scalding-db/src/main/scala/com/twitter/scalding/db/macros/impl/handler/BlobTypeHandler.scala

```diff
@@ -0,0 +1,20 @@
+package com.twitter.scalding.db.macros.impl.handler
+
+import com.twitter.scalding.db.macros.impl.FieldName
+import scala.reflect.macros.Context
+import scala.util.Success
+
+object BlobTypeHandler {
+  def apply[T](c: Context)(implicit
+    accessorTree: List[c.universe.MethodSymbol],
+    fieldName: FieldName,
+    defaultValue: Option[c.Expr[String]],
+    annotationInfo: List[(c.universe.Type, Option[Int])],
+    nullable: Boolean): scala.util.Try[List[ColumnFormat[c.type]]] = {
+
+    assert(defaultValue.isEmpty)
+    assert(annotationInfo.isEmpty)
```

Comment:
why use `assert` here but return `Success`?

Why not:
```scala
if (defaultValue.nonEmpty || annotationInfo.nonEmpty) Failure(new Exception(s"invalid args: defaultValue = $defaultValue annotationInfo = $annotationInfo"))
else Success(List(ColumnFormat(c)(accessorTree, "BLOB", None)))
```

Reply:
Just changed it to `required` as was done in the original PR, would that be ok of `if else` better?

Reply:
I will do if else of Success Failure makes more sense that throwing

Reply:
well, to a method that returns a `Try` shouldn't also throw.

Reply:
Done